deploy-agent/deployd/common/utils.py

```diff
@@ -267,7 +268,27 @@ def get_container_health_info(commit, service, redeploy) -> Optional[str]:
                                             tags={"status": "healthy", "service": service, "commit": commit})
             return returnValue
         else:
-            return None
+            fn = os.path.join("/mnt/deployd/", "{}_HEALTH_CHECK".format(service))
+            if not os.path.isfile(fn):
+                return None
+        
+            with open(fn, 'r') as f:
+                healthcheckConfigs = dict((n.strip('\"\n\' ') for n in line.split("=", 1)) for line in f)
```

Comment:
why don't you store as a json object so you don't need to parse like this?

Reply:
This is quite straightforward and maintains the same format as the script configurations.